packages/vanilla/src/components/pagination/pagination.stories.js

```diff
@@ -0,0 +1,40 @@
+export default {
+  title: 'Patterns/Pagination'
+};
+
+const Template = () => {
+  return `
+    <div class="cbp-pagination">
+      <div class="cbp-pagination__items">
+        <select aria-label="Select Item Amount">
+          <option value="">10/Page</option>
+          <option value="25">25 Per Page</option>
+          <option value="50">50 Per Page</option>
+          <option value="100">100 Per Page</option>
+          <option value="100">9000 Per Page</option>
+        </select>
+        <span class="cbp-pagination__item-count">
+          1-10 of 103 Items
+        </span>
+      </div>
+      <div class="cbp-pagination__pages">
+        <button class="cbp-pagination__prev" type="button" aria-label="Previous Page">
+          <i class="fas fa-chevron-left"></i>
+        </button>
+        <select aria-label="Select Page">
+          <option value="1">Pg. 1 of 1000</option>
```

Comment:
I don't think abbreviating Page as "Pg." is buying us anything for the cognitive burden it creates (especially in screen readers). It's saving so little space, I'd just spell it out, if it's needed at all.